lib/index.d.ts

```diff
@@ -269,15 +269,26 @@ declare namespace Stardog {
             function get(conn: Connection, database: string, params?: object): Promise<HTTP.Body>;
 
             /** 
-             * Sets the integrity constraints on a given database. 
+             * Adds integrity constraints to a given database. 
              * 
              * @param {Connection} conn the Stardog server connection
              * @param {string} database the name of the database
              * @param {string} icvAxioms an RDF block containing the axioms to be added
              * @param {object} options an object specifying the contentType of the RDF data. Default: text/turtle
              * @param {object} params additional parameters if needed
              */
-            function set(conn: Connection, database: string, icvAxioms: string, options?: { contentType: HTTP.ContentMimeTypes }, params?: object): Promise<HTTP.Body>;
+            function add(conn: Connection, database: string, icvAxioms: string, options?: { contentType: HTTP.ContentMimeTypes }, params?: object): Promise<HTTP.Body>;
+
+            /** 
+             * Removes integrity constraints from a given database. 
+             * 
+             * @param {Connection} conn the Stardog server connection
+             * @param {string} database the name of the database
+             * @param {string} icvAxioms an RDF block containing the axioms to be removed
+             * @param {object} options an object specifying the contentType of the RDF data. Default: text/turtle
+             * @param {object} params additional parameters if needed
```

Comment:
You may want to add a note about the `contentType` has to "match" or make sense or whatever with regard to the `icvAxioms` parameter.